iFixit/react-commerce
# Avoid long text make compatibility image shrink

closes #1775 

### QA

1. Visit Vercel preview
2. Verify that even with long variant names the variant image is not going to shrink anymore
